packages/test-dev-server/src/dev-server.ts

```diff
@@ -148,7 +148,7 @@ class DevServer {
           if (this.#devOptions?.platform === 'browser') {
             // TODO: send reload message to client
           }
-          this.#devEngine?.ensureLatestBuild();
+          this.#devEngine?.ensureCurrentBuildFinish();
```

Comment:
Shouldn't this be `ensureLatestBuild` for `eagerRebuild: false`? For `eagerRebuild: true`, I think this line is not needed. (IIUC the default value of `eagerRebuild` is false)